src/components/pages/faq/introduction.tsx

```diff
@@ -128,8 +221,53 @@ const Introduction: React.FC = () => {
         },
         {
           title: 'How do I deploy a PayID server?',
-          description:
-            'PayID is designed by devs for devs. You can deploy a PayID server with just a few commands with existing web infrastructure, and then integrate it into an application or account system in just a few lines of code.',
+          description: (
+            <>
+              <p>
+                PayID is designed by devs for devs. You can deploy a PayID
+                server with just a few commands with existing web
+                infrastructure, and then integrate it into an application or
+                account system with just a few lines of code.
+              </p>
+              <br />
```

Comment:
Why is BR needed here also?

Reply:
It creates a paragraph break in the text, without it the 2 paragraphs run together.

Reply:
But I see your point, br shouldn't be used for presentation. Just bad habit.